### PR TITLE
Projects

### PR DESCRIPTION
**Pull request by dimaverh**

### CODE REVIEW

File: src/components/account/accountFormModal.tsx [hunk 1]
```diff
@@ -41,6 +41,20 @@ export default class AccountFormModal extends React.Component<IProps, any> {
...
+        if (xhr.readyState === XMLHttpRequest.DONE && xhr.status === 200) {
+          const img = new Image();
+          img.crossOrigin = 'Anonymous';
+          img.src = URL.createObjectURL(xhr.response); // create <img> with src set to the blob
+          this.setState({src: img});
+        }
+      };
+      xhr.open('GET', this.props.account.avatar.url + '?cacheblock=true', true);
+      xhr.send();
+    }
```
Comment (sergeyq) [inline src/components/account/accountFormModal.tsx]: Xhr is Promise,  fetch is more better way than xhr (fetch.then().catch()). need to detect errors by async queries. (I like to use Async functions with operator await)

File: src/components/helpers/fields/uploadField/index.tsx [hunk 2]
```diff
@@ -11,6 +11,7 @@ interface IProps {
   field: any;
   type: string;
   name: string;
+  preview: any;
```
Comment (sergeyq) [inline src/components/helpers/fields/uploadField/index.tsx]: try to avoid type of any

File: src/components/helpers/fields/uploadField/index.tsx [hunk 3]
```diff
@@ -19,12 +20,12 @@ interface IState {
   src: string;
 }
 
-const src: string = './example.jpg';
+let src: string = './example.jpg';
```
Comment (sergeyq) [inline src/components/helpers/fields/uploadField/index.tsx]: u can set default value in const InitialState

File: src/components/helpers/fields/uploadField/index.tsx [hunk 4]
```diff
@@ -35,25 +36,19 @@ export default class UploadText extends React.Component<IProps, IState> {
...
         <div onChange={this.uploadFile}>
           <Avatar
             width={300}
             height={295}
             onCrop={this.onCrop}
             onClose={this.onClose}
-            onFileLoad={this.onBeforeFileLoad}
-            src={this.props.field.value.url}
-	          // img={document.createElement('img')}
+            // onFileLoad={this.onBeforeFileLoad}
```
Comment (sergeyq) [inline src/components/helpers/fields/uploadField/index.tsx]: remove comments

File: src/components/account/accountFormModal.tsx (hunk 1 above)
Comment (DmitryLeviossa) [reply src/components/account/accountFormModal.tsx]: You shouldn't use fetch at all, use sagas

File: src/components/helpers/fields/uploadField/index.tsx (hunk 3 above)
Comment (DmitryLeviossa) [reply src/components/helpers/fields/uploadField/index.tsx]: As I understood that variable not used, you have same key in state

File: src/components/management/projects/project/assignFormModal/index.tsx [hunk 5]
```diff
@@ -155,7 +155,6 @@ export default class AssignFormModal extends React.Component<IProps, any> {
     const { users, price } = this.state.form;
     e.preventDefault();
     const { selectedAssignUser } = this.state;
-    console.log(selectedAssignUser.assign_user_id);
     // if (this.validate({
```
Comment (DmitryLeviossa) [inline src/components/management/projects/project/assignFormModal/index.tsx]: Remove commented code

**Pull request closed without merging**